docs/windows.md

```diff
@@ -5,10 +5,31 @@ users should be aware of.
 
 ## Line endings are not converted
 
-Jujutsu does not honor `.gitattributes` and does not have a setting like Git's
-`core.autocrlf`. This means that line endings will be checked out exactly as
-they are committed and committed exactly as authored. This is true on all
-platforms, but Windows users are most likely to miss CRLF conversion.
+Jujutsu does not currently honor `.gitattributes` and does not have a setting
+like Git's `core.autocrlf`. This means that line endings will be checked out
+exactly as they are committed and committed exactly as authored. This is true on
+all platforms, but Windows users are most likely to miss CRLF conversion.
+
+If you Git repository expects Windows users to have `core.autocrlf` set to
```

Comment:
Typo: "If you" => "If your Git repository ..."

Reply:
Whoops, fixed.